firestore/cloud-client/query_filter_or.py

```diff
@@ -12,28 +12,51 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# [START firestore_query_filter_or]
 from google.cloud import firestore
-from google.cloud.firestore_v1.base_query import FieldFilter, Or
 
 
-def query_or_composite_filter(project_id: str) -> None:
+def query_or_filter(project_id: str) -> None:
     # Instantiate the Firestore client
     client = firestore.Client(project=project_id)
```

Comment:
To make the sample copy-paste runnable as per the [style guide](https://googlecloudplatform.github.io/samples-style-guide/), the client creation should be included in the code sample.

Same with other samples.

Reply:
Yeah, this is in line with the other languages: https://firebase.google.com/docs/firestore/query-data/queries#or_queries